include/kernel.h

```diff
@@ -773,7 +773,7 @@ extern void k_thread_foreach_unlocked(
  * @param p3 3rd entry point parameter.
  * @param prio Thread priority.
  * @param options Thread options.
- * @param delay Scheduling delay (in milliseconds), or K_NO_WAIT (for no delay).
+ * @param delay Scheduling delay, or K_NO_WAIT (for no delay).
```

Comment:
im missing a place where timeout argument is well described. `k_timeout_t` in `sys_clock.h` is missing description at all and in the api it is very limited. I know that it is because of two modes but still maybe there should be something like: `see k_timeout_t for more details`.

Reply:
You're right, there should be a doc block on k_timeout_t.  Will add one.